From: KevinDuan <duankaiwen@kylinos.com>
Date: Wed, 5 Jun 2024 03:31:38 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 23.2% (17 of 73 strings)

Translation: openkylin-nile-new/kylin-os-manager-replacement
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-replacement/ug/
---
 .../replacement/translations/replacement_ug.ts         | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/treasure-chest/replacement/translations/replacement_ug.ts b/treasure-chest/replacement/translations/replacement_ug.ts
index a7de687..8d7b334 100644
--- a/treasure-chest/replacement/translations/replacement_ug.ts
+++ b/treasure-chest/replacement/translations/replacement_ug.ts
@@ -8,13 +8,13 @@
         <location filename="../view/acceptmigrationdata.cpp" line="201"/>
         <location filename="../view/acceptmigrationdata.cpp" line="204"/>
         <source>sec</source>
-        <translation type="unfinished"></translation>
+        <translation>سېكونت</translation>
     </message>
     <message>
         <location filename="../view/acceptmigrationdata.cpp" line="201"/>
         <location filename="../view/acceptmigrationdata.cpp" line="204"/>
         <source>min</source>
-        <translation>مىنۇت</translation>
+        <translation>ئەڭ قىسقا</translation>
     </message>
     <message>
         <location filename="../view/acceptmigrationdata.cpp" line="204"/>
@@ -50,7 +50,7 @@
     <message>
         <location filename="../globaldata.cpp" line="139"/>
         <source>Application</source>
-        <translation>ئەپ</translation>
+        <translation>ئىلتىماس قىلىش</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="140"/>
@@ -65,7 +65,7 @@
     <message>
         <location filename="../globaldata.cpp" line="146"/>
         <source>Start</source>
-        <translation>باشلان</translation>
+        <translation>قوزغىتىش</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="147"/>
@@ -155,7 +155,7 @@
     <message>
         <location filename="../globaldata.cpp" line="168"/>
         <source>Share</source>
-        <translation>ئېكراننى ھەمبەھرىلەش</translation>
+        <translation>تەڭ بەھرىمەن بولۇش</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="169"/>
@@ -180,7 +180,7 @@
     <message>
         <location filename="../globaldata.cpp" line="175"/>
         <source>Range</source>
-        <translation type="unfinished"></translation>
+        <translation>دائىرە</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="176"/>
@@ -215,7 +215,7 @@
     <message>
         <location filename="../globaldata.cpp" line="184"/>
         <source>Required</source>
-        <translation>زۆرۈر</translation>
+        <translation>چوقۇم  تولدۇرۇش</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="185"/>
@@ -305,7 +305,7 @@
     <message>
         <location filename="../globaldata.cpp" line="212"/>
         <source>Reboot</source>
-        <translation>دەرھال قايتا قوزغىتىش</translation>
+        <translation>قايتا قوزغىتىش</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="213"/>
@@ -345,7 +345,7 @@
     <message>
         <location filename="../globaldata.cpp" line="222"/>
         <source>Ok</source>
-        <translation>جەزملەشتۈرۈش</translation>
+        <translation>ماقۇل</translation>
     </message>
     <message>
         <location filename="../globaldata.cpp" line="223"/>
